microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md

```diff
@@ -0,0 +1,176 @@
+---
+title: What's new in DevTools (Microsoft Edge 99)
+description: Host source maps on Azure DevOps Symbol Server to debug your original source code securely.  Use source maps to unminify performance profiles.  DevTools extension now for Microsoft Visual Studio.  3D View tool replaces Layers tool.  Improved accessibility in Network Console and 3D View tools.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 03/04/2022
+---
+# What's New in DevTools (Microsoft Edge 99)
+
+The following sections list the announcements from the Microsoft Edge DevTools team.  To try the latest features of DevTools and the Microsoft Edge DevTools extension for Visual Studio Code, read these announcements.  To stay up to date with the latest and greatest features in your developer tools, download the [Microsoft Edge preview channels](https://www.microsoftedgeinsider.com/download) and [follow the Microsoft Edge DevTools team on Twitter](https://twitter.com/EdgeDevTools).
+
+If you are on Windows, Linux, or macOS, consider using the Canary preview channel of Microsoft Edge as your default development browser.  The preview channels give you access to the latest features of Microsoft Edge DevTools.
+
```

Comment:
This would also be a good opportunity to promote the new repo: 

"If you want to report a problem with DevTools or ask for a new feature, you can [file an issue](https://github.com/MicrosoftEdge/DevTools/issues/new/choose) on the [GitHub Edge DevTools repository](https://github.com/MicrosoftEdge/DevTools)."